errors.yaml

```
@@ -48,12 +52,24 @@ errors:
     description: "The Content-Type header does not match the request body payload format or the format is invalid."
   - code: missing_payload
     description: "The Content-Type header was specified, but no request body was sent to the server or the request body is empty."  
-  - code: dump_already_in_progress
-    description: "A dump creation is already in progress and a new one can't be triggered until the previous dump creation is not finished."
+  - code: dump_already_processing
+    description: "Dump creation is already in progress. A new dump creation process can be triggered when the current one is complete."
+  - code: dump_not_found
+    description: "The requested dump could not be found."  
   - code: dump_process_failed
     description: "An error occurred during dump creation process, task aborted."
   - code: invalid_ranking_rule
-    description: "The provided ranking rules are invalid. For more information on ranking rule configuration, check our [relevancy guide](https://docs.meilisearch.com/learn/core_concepts/relevancy.html#ranking-rules)"
+    description: "The provided ranking rules are invalid. This may be due to syntax errors in your ranking rules or specifying custom ranking rules on reserved fields or reserved expressions. For more information on ranking rule configuration, check our [relevancy guide](https://docs.meilisearch.com/learn/core_concepts/relevancy.html#ranking-rules)."
   - code: invalid_geo_field
     description: "The provided `_geo` field of one or more documents is invalid. Read more about `_geo` and how to troubleshoot it in [our dedicated guide](https://docs.meilisearch.com/reference/features/geosearch.html)."
+  - code: invalid_api_key
+    description: "The requested resources are protected with an API key. The provided API key is invalid. Read more about it at in our [dedicated guide](https://docs.meilisearch.com/reference/features/authentication.html)."
+  - code: invalid_store_file
+    description: "The `data.ms` folder is in an invalid state. Your `.mdb` file is corrupted or the `data.ms` folder has been replaced by a file."
+  - code: no_space_left_on_device
+    description: "The host system partition has reached its maximum capacity and can no longer accept writes." 
+  - code: database_size_limit_reached
+    description: "The requested database has reached its maximum limit; no further documents can be added. For information on customizing the maximum database size accepted by MeiliSearch, check out this [guide](https://docs.meilisearch.com/reference/features/configuration.html#max-udb-size)."
```

I know it's on the error code, but perhaps it's good to specify it's the size limit we're talking about?

```suggestion
    description: "The requested database has reached its maximum size; no further documents can be added. For information on customizing the maximum database size accepted by MeiliSearch, check out this [guide](https://docs.meilisearch.com/reference/features/configuration.html#max-udb-size)."
```